spec/features/charity_admin_creates_loan_spec.rb

```diff
@@ -24,7 +24,6 @@
     click_on "Create Loan"
 
     expect(current_path).to eq(admin_families_path)
-    # expect(page).to have_content("1000.00")
-    # expect(page).to have_content("This is a purpose")
+    expect(page).to have_content("Loan Created Successfully")
```

Comment:
Good change. Maybe we should also add a within CSS item? Like within the css class success